### PR TITLE
📑 Multiple file upload

### PR DESCRIPTION
**Pull request by mermop**
[this trello card](https://trello.com/c/jMp8pK7B/88-upload-multiple-pdf-word-submissions)

### CODE REVIEW

**Comment by mermop** [comment]
I've separated out preventing duplicate file uploads into another story https://trello.com/c/dARy9NwO/117-prevent-duplicate-submission-uploads

File: app/controllers/submissions_controller.rb [hunk 1]
```diff
@@ -76,4 +73,44 @@ def create_submission_params
...
+    @submission = Submission.new(create_submission_params)
+    @submission.file.attach(create_submission_params[:file])
+    if @submission.save
+      redirect_to @submission, notice: "Submission was successfully created."
+    else
+      render :new
+    end
+  end
+
+  def create_multiple_submissions # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```
Comment (Rabid-Dan) [inline app/controllers/submissions_controller.rb]: This comment is probably more valid in the new year but its maybe worth pushing a bunch of this logic out into a service object that handles both single and multiple submission creation.

**Comment by Rabid-Dan** [comment]
feels like its probably worth putting a couple of tests in here, even if its just for creating 1 vs creating many submissions

File: app/controllers/submissions_controller.rb (hunk 1 above)
Comment (mermop) [reply app/controllers/submissions_controller.rb]: Good idea, makes it much easier to test 👌

File: app/views/submissions/show.html.erb [hunk 2]
```diff
@@ -29,16 +29,19 @@
...
     <div>
-      <% if @submission.can_process? %>
-        <h3>Submission text</h3>
-        <%= link_to('Edit', edit_submission_path(@submission), class: 'btn btn-primary') %>
-        <%= link_to('Mark as ready', process_submission_path(@submission), class: 'btn btn-success',
-            method: :put, data: { confirm: 'Are you sure? This will prevent further edits to the text.' }) %>
-      <% elsif @submission.text.present? %>
+      <% if @submission.text.present? %>
         <h3>Submission text</h3>
+        <% if @submission.can_process? %>
```
Comment (mermop) [inline app/views/submissions/show.html.erb]: this is actually a bit separate - and probably a testament to the need for tests when there's this much branching in the UI

**Pull request merged**